qa/integration-tests/src/test/java/io/camunda/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -718,6 +719,28 @@ public SnapshotId waitForSnapshotAtBroker(final Broker broker) {
     return waitForNewSnapshotAtBroker(broker, null);
   }
 
+  public void triggerAndWaitForSnapshots() {
+    // Ensure that the exporter positions are distributed to the followers
+    getClock().addTime(Duration.ofSeconds(15));
```

Comment:
I guess this is not required in 1.1. This has changed in 1.2 as part of building state on followers.

Reply:
Removed this from the source code.